src/version-2-1-0.md

```diff
@@ -6,25 +6,26 @@ Category: news
 
 Please check out our [release notes] for version 2.1.0.
 
-We've added new APIs for listing the subscriptions of a subscriber and for
-showing the configurations of the push service providers for services that
+New APIs have been added: one for listing the subscriptions of a subscriber and
+another for showing the configurations of the push service providers for services that
 exist.
 
 Concurrency issues in ADM and APNS have been fixed. The APNS implementation
 has been changed to use a new, reliable worker pool implementation.
 
-This implementation has been running at production scale at if(we) serving push
-notifications for Tagged/hi5 for a while now and has been proven to be
-reliable. We have 12 processes handling a peak of 60 APNS push notifications
-per second, and the tier is overprovisioned in order to be able to deal with
+This implementation has been running at production scale, at a company named
+*if(we)*, and sending push notifications for the Tagged and hi5 social networks
```

Comment:
really long sentence.

"and sending push notifications" -> "sending push notifications" ? Without the parenthetical comment, the phrasing would be awkward: "This implementation has been running at production scale, <s>and</s> sending push notifications for the Tagged and hi5 social networks"


"has been running reliably at production scale", remove the "and has proven to be reliable"